packages/itwinui-css/package.json

```diff
@@ -36,19 +33,19 @@
     "npm-run-all": "^4.1.5",
     "postcss": "^8.4.5",
     "postcss-discard-comments": "^5.0.1",
-    "postcss-scss": "^4.0.3",
+    "postcss-scss": "^4.0.4",
+    "sass": "^1.54.4",
```

Comment:
Why not `sass-embeded`?

Reply:
Oh, I see it it left bellow. Then wtf 😁

Reply:
`sass` is used for the `-watch` flag in `scripts`
`sass-embedded` is used for the build (in `generateCss.js`)

Reply:
Can't you use `sass-embeded` for both?

Reply:
`sass-embedded` doesn't come with a CLI or watch flag